Exercise_6/ReversePolishNotation.js

```diff
@@ -0,0 +1,44 @@
+// const process = require('process');
+// console.log(calculate(process.argv[2]));
+
+function calculate(argument,extraOperators) {
+    const invalidArgumentError = new Error('The argument ' + argument + ' is invalid');
+    const signs = argument.split(' ');
+    const stack=[];
```

Comment:
Spaces. `const stack = [];`. And actually, you don't need this variable because you have the initial state in the reduce function.